src/hotspot/share/gc/shenandoah/shenandoahConcurrentGC.cpp

```diff
@@ -242,12 +242,16 @@ bool ShenandoahConcurrentGC::collect(GCCause::Cause cause) {
       heap->set_old_evac_reserve(0);
       heap->reset_old_evac_expended();
       heap->set_promoted_reserve(0);
+      log_info(gc, ergo)("At end of Concurrent GC, old_available: " SIZE_FORMAT "%s out of total: " SIZE_FORMAT "%s,"
```

Comment:
This looks a lot like the implementation of `ShenandoahGeneration::log_status` . Could consolidate these messages and reduce logging duplicate information.